lib/aggregation/accumulator/src/index.js

```diff
@@ -351,9 +351,13 @@ const runCLI = () => {
 
   // Create the app, replay any failed events, and listen
   const app = accumulator();
-  dataflow.replay(app.reducer, 0, () => {
-    app.listen();
-  });
+  app.listen();
+  if (cluster.isWorker()) {
+    debug('Replaying for accumulator');
```

Comment:
Debug already logs the abacus-accumulator location, so we might simply log something like `Starting replay...`